app/code/Magento/Quote/Model/GuestCart/GuestCartRepository.php

```diff
@@ -6,7 +6,6 @@
 namespace Magento\Quote\Model\GuestCart;
 
 use Magento\Quote\Api\GuestCartRepositoryInterface;
-use Magento\Quote\Model\QuoteIdMask;
```

Comment:
We can't remove this line because `Magento\Quote\Model\QuoteIdMask` used in `get` method without the fully qualified name